.github/workflows/CD_release_npm.yml

```diff
@@ -31,10 +31,13 @@ jobs:
 
       - run: |
           yarn install
-          yarn build
+          yarn build-mjml-react
 
       - name: Run semantic release (release + bump version)
-        run: npx semantic-release
+        run: |
+          cd dist
+          npx semantic-release
         env:
+          HUSKY: 0
```

Comment:
Need to skip husky pre-commit hook since it causes an error when running semantic-release from the dist folder